web/Web/web/CAL/views.py

```diff
@@ -198,3 +199,70 @@ def get_ajax(self, request, *args, **kwargs):
         except CALError as e:
             # TODO: add proper http response for CAL errors
             return JsonResponse({"message": "Ops! CALError."}, status=404)
+
+
+class DSLoggingView(views.CsrfExemptMixin,
+                    views.LoginRequiredMixin,
+                    RetrievalMethodPermissionMixin,
+                    views.JsonRequestResponseMixin,
+                    generic.View):
+    """
+    View to log info of Dynamic Sampling
```

Comment:
`View to log stratum information in the dynamic sampling method`

Reply:
Fixed